app/gui/view/component-browser/component-group-view/src/lib.rs

```diff
@@ -0,0 +1,224 @@
+//! An EnsoGL implementation of a Component Group View.
+//!
+//! A Component Group View is displayed as a header (containing the name of a group of Enso
+//! components) and a list of entries below it (names of the components belonging to that group).
```

Comment:
We try not to repeat the names in the docs. The names such as "Enso" or "EnsoGL" do not have to appear here. It makes it easier in the future to refactor these names. Also:

1. "An EnsoGL implementation of a Component Group View." – this sounds like there was a non-EnsoGL implementation of "Component Group View". If so, where it is? Also, the first paragraph should explain what this file is about, but it does not. After reading it, I don't know what "Component Group View" really is.
2. "header (containing the name of a group of Enso components)", what else the header contains?
3. "[...] it (names of the components belonging to that group)", this sounds like the explanation is connected with "it", but it is not.
4. This docs should be linked to our design document.